parts/linux/cloud-init/artifacts/labels.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update Labels for Kubernetes nodes
+After=kubelet.service
+[Service]
+Restart=always
+RestartSec=300
```

Comment:
how do we decide this value? 5 min seems too long

Reply:
ok, I take it back. We do want to make sure this doesn't run too frequent to drain api-server resource while also don't want to be too slow in reconciling the value.

Reply:
I had it at 1 minute before, but we were concerned about large clusters

Reply:
After talking offline about this I assume the 5 minute window is ok? :)

Reply:
changed to 30 seconds with retry on failure